jupyter/datascience/ubi9-python-3.12/utils/pipeline_app.patch

```diff
@@ -0,0 +1,12 @@
+--- a/pipeline_app.py	2023-11-09 13:36:35
++++ b/pipeline_app.py	2023-11-09 12:10:35
+@@ -296,7 +296,8 @@
+         _validate_pipeline_definition(pipeline_definition)
+     except Exception:
+         raise click.ClickException("Pipeline validation FAILED.")
+-
++    
```

Comment:
hehe, don't like this change, but it's same as we already have in our main branch, so... leaving this up to you whether you will update or not